x-pack/plugins/serverless_search/public/application/components/web_crawlers/elastic_managed_web_crawler_coming_soon.tsx

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiIcon,
+  EuiTitle,
+  EuiText,
+  EuiBadge,
+  EuiStepsHorizontal,
+  EuiStepsHorizontalProps,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { css } from '@emotion/react';
+import { useKibanaServices } from '../../hooks/use_kibana';
+import { useAssetBasePath } from '../../hooks/use_asset_base_path';
+
+// import { FeedbackForm } from '../coming_soon/feedback_form';
+
+import { BACK_LABEL } from '../../../../common/i18n_string';
+
+export const ElasticManagedWebCrawlersCommingSoon: React.FC = () => {
```

Comment:
A ton of stuff in here is basically the same as in the other connectors pages. You can extract the common things to a common React component, and then have much smaller components filling in the interior stuff that's different.

Reply:
For now we have these shared content components:
`x-pack/plugins/serverless_search/public/application/components/common/decorative_horizontal_stepper.tsx`
`x-pack/plugins/serverless_search/public/application/components/connectors/connector_icon.tsx`

Reply:
That's not quite what I mean. The entire structure of this page is identical to the connectors coming soon. It would be better to extract that structure to a common component, rather than (just) extracting the children.

We don't need to do this now in this pull request, but keep that in mind for the future. If you find yourself doing a lot of the same stuff in multiple places, it usually means you can share code.